AtRuleFeatureDetection/explainer.md

```diff
@@ -0,0 +1,278 @@
+of evaluating the query.
+
+Suppose an author instead wants to condition part of their stylesheet on whether an implementation recognizes 
+`fancy-display` *at all*. Implementations that recognize `fancy-display`, and implementations that don't, both 
+must at least parse the above media query. If feature detection were determined purely based on whether the 
+at-rule parses successfully, it would not be possible to feature-detect support for `fancy-display`. The 
+exception we carve out allows us to handle this situation: Implementations that recognize `fancy-display` will 
+parse it on their "known media features" path, and implementations that don't recognize it will parse it on their 
+"forgiving catch-all" path.
+
+```css
+/* This @supports query returns true if the implementation knows what a fancy-display is. */
+@supports at-rule(@media (fancy-display)) {
+
+    /* This @media query returns true if the user is actually using a fancy-display. */
+    @media (max-width: 800px and fancy-display) {
+        /* ... */
+    }
+
+    /* This @media query returns true if the user is detectably not using a fancy-display. */
+    @media (max-width: 800px and not(fancy-display)) {
+        /* ... */
+    }
+}
+```
+
+### Detect whether a given declaration is supported within an at-rule block
+
+This form allows testing support for descriptors or property declarations within a given at-rule.
+Given an at-rule name and a declaration, the support query returns true if the declaration parses within
+the at-rule's block, false otherwise.
+
+```css
+@supports at-rule(@property; syntax: "<color>") {
+    /* declare custom properties with color syntax */
+}
+
+@supports at-rule(@position-try; box-shadow: 0px 0px) {
+    /* declare rules that set different box-shadow locations for different position-areas */
+}
+```
+
+Such tests could also be accomplished with the previous form, but this form allows authors to omit the prelude
+and/or required declarations that are not the subject of the test.
+For example, the `@font-face` rule requires `font-family` and `src` descriptors.
+An author who wants to feature detect support for `font-feature-settings` using the previous form would need
+to supply dummy values for `font-family` and `src` in order for the test block to parse successfully.
+The query would thus be quite verbose:
+
+```css
+/* Testing as a full at-rule using the previous form */
+@supports at-rule(@font-face {font-family: test; src: local(test); font-feature-settings: "hwid"} ) {
+    ...
+}
+```
+
+With this form, the author can simplify to:
+
+```css
+/* Simpler query using this form */
+@supports at-rule(@font-face; font-feature-settings: "hwid") {
+    ...
+}
+```
+
+## Accessibility, Privacy, and Security Considerations
+
+No accessibility, privacy, or security considerations have been identified for this feature.
+
+## Stakeholder Feedback / Opposition

```

Comment:
nit: I would guess this section can be removed altogether

Reply:
It's part of the TAG's most recent [explainer template](https://github.com/w3ctag/explainer-explainer/blob/main/template.md); I wanted to leave it in as a reminder to add links to position threads once we send an intent.